pkg/server/handler_test.go

```diff
@@ -0,0 +1,309 @@
+package server
+
+import (
+	"testing"
+
+	"github.com/CESARBR/knot-babeltower/pkg/logging"
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/controllers"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/mocks"
+)
+
+func TestMsgHandler_Start(t *testing.T) {
+	type fields struct {
+		logger          logging.Logger
+		amqp            network.AmqpReceiver
+		thingController controllers.ThingController
+	}
+	type args struct {
+		started chan bool
+		msgChan chan network.InMsg
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		wantErr bool
+	}{
+		{
+			"when started channel nil start should return error",
```

Comment:
As you choose to describe the tests in a behaviour style, I think it'd good avoid putting details in the messages.
For example, maybe it could be "when started channel not passed/provided should return error" or "should return an error when started channel not passed/provided". Also, I don't think we need to mention the `start` operation explicitly since the SUT (system under testing) is already clear by the context: **`mc.start()`.**

Reply:
In alignment with this comment, I suggest the following instead (changed @netoax's suggestion a little): :point_down: 

- _"when started channel is not passed/provided, and error should be returned"_ **or** _"an error should be returned when started channel is not passed/provided"_

Reply:
Done